py-rse/git-advanced.Rmd

```diff
@@ -308,8 +312,8 @@ def plot_fit(curve_xmin, curve_xmax, max_rank, beta, ax):
     ax.loglog(xvals, yvals, color='grey')
 ```
 
-where the maximum word frequency rank corresponds to intercept
-in our linear form, and \(-\beta + 1\) the slope.
+where the maximum word frequency rank corresponds to \(c\), 
+and \(-\beta + 1\) the exponent in the power law.
```

Comment:
Is this where you were suggesting in the previous PR that we should use -1 / alpha instead of -beta + 1?

Reply:
Yeah, that's what I was thinking.  My python is bad enough that I wasn't that confident I'd make all the right changes though...

Reply:
Cool - I've gone through and switched everything to alpha.